example/src/RadioButtonExample.js

```diff
@@ -53,9 +53,10 @@ const RadioButtonGroupExample = ({ theme }) => {
           onValueChange={handleSelect}
           value={selected}
         >
-          <RadioButtonRow label="First" value="1" />
-          <RadioButtonRow label="Second" value="2" />
-          <RadioButtonRow label="Third" value="3" />
+          <RadioButtonRow label="Zero" value={0} />
+          <RadioButtonRow label="1st" value={1} />
+          <RadioButtonRow label="2nd" value={2} />
+          <RadioButtonRow label="3rd" value={3} />
```

Comment:
since you're using string | number, can you set one of these examples to use a string?

Reply:
@peterpme Done.